hat/app/org/hatdex/hat/resourceManagement/actors/HatServerProviderActor.scala

```diff
@@ -50,25 +51,25 @@ class HatServerProviderActor @Inject() (
   import HatServerProviderActor._
 
   private val activeServers = mutable.HashMap[String, ActorRef]()
-  private implicit val hatServerTimeout: Timeout =
+  implicit private val hatServerTimeout: Timeout =
     configuration.get[FiniteDuration](
       "resourceManagement.serverProvisioningTimeout"
     )
 
   def receive: Receive = {
     case HatServerRetrieve(hat) =>
-      log.debug(s"Retrieve HAT server $hat for $sender")
+      log.info(s"Retrieve HAT server $hat for $sender")
       val retrievingSender = sender
       getHatServerActor(hat) map { hatServerActor =>
-        log.debug(
-          s"Got HAT server provider actor, forwarding retrieval message with sender $sender $retrievingSender"
+        log.info(
+          s"Success: Got HAT server provider actor, forwarding retrieval message with sender $sender $retrievingSender"
         )
         hatServerActor tell (HatServerActor.HatRetrieve(), retrievingSender)
       } onComplete {
         case Success(_) => ()
         case Failure(e) =>
           log.warn(
-            s"Error while getting HAT server provider actor: ${e.getMessage}"
+            s"Failure: Error while getting HAT server provider actor: ${e.getMessage}"
```

Comment:
after our chat this morning i wonder if a stack trace would also be usefull ?

Reply:
Sure.